src/popup.html

```diff
@@ -80,6 +86,81 @@ <h4 class="i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></h4>
             </div>            
         </sign-up-panel>
 
+        <!-- Panel: Phone Masks (Secondary to Email masks) -->
+        <phone-masks-panel data-panel-id="phone-masks" class="is-hidden fx-relay-panel" id="phone-masks-panel">
+          <div class="fx-relay-panel-content">
+            
+            <div class="fx-relay-phone-default-view is-hidden">
+              <div class="fx-relay-phone-container"> 
+                  <p id="fx-relay-user-phone-country">
+                    <img class="i18n-alt-tag" alt="" src="/icons/us-flag.svg">
```

Comment:
This is causing an error in i18n.js since anything with `class="i18n-alt-tag"` needs the following attribute, `data-i18n-message-id=<your id>`

Reply:
Removing this as its only for decoration.